File: src/algowithjava/programmers/level1/Watermelon.java
package algowithjava.programmers.level1;


// 수박수박수박수
public class Watermelon {
    public String solution(int n) {
        String answer = "";

        for(int i=1; i<n+1; i++){
            if(i%2!=0){
                answer += "수";
            }else{
                answer += "박";
            }
        }
        return answer;
    }
}

File: src/algowithkotlin/programmers/level1/Watermelon.kt
class Watermelon {
    fun solution(n: Int): String {
        var answer = ""
        
        for(i in 1..n){
          when{
            i%2!=0 ->
              answer += "수"
            else ->
              answer += "박"
          }
        }
        
        return answer
    }
}